data/sql/updates/pending_db_world/rev_1609174873784850500.sql

```diff
@@ -0,0 +1,29 @@
+-- INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1609174873784850500');
```

Comment:
Uncomment back this line, as mentioned in https://www.azerothcore.org/wiki/How-to-create-a-PR it's a needed line by db-assembler. You probably commented for testing purposes.

Reply:
Yes, forgot this from testing, thanks!